frontend/app/src/scenes/Home/styles.css

```diff
@@ -39,6 +39,10 @@
     margin-top: 12px;
 }
 
+:local .formSelect {
```

Comment:
remove `:local`

Reply:
from the whole sheet?

Reply:
yeah it's better to remove all. We don't need it. It's `:local` by default.

Reply:
okay